tests/storage/sdm_add_bitmap_test.py

```diff
@@ -28,6 +28,7 @@
 from vdsm.storage import exception as se
 from vdsm.storage import guarded
 from vdsm.storage import qemuimg
+from vdsm.storage.volume import Qcow2BitmapInfo
 from vdsm.storage.sdm import volume_info
```

Comment:
We try to keep alphabetical order within import blocks.

Could you please move the `volume` import below `sdm` (and above `sdm.api`)?

Reply:
On it :)